doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -90,14 +129,16 @@ To perform a FOTA update, complete the following steps:
 
    |fota_upgrades_building|
 
-#. Download the :file:`app_update.bin` image file to your device.
+#. Download the :file:`dfu_application.zip` zip archive or the :file:`app_update.bin` image file to your device.
+   What file should be chosen depends on the application used to perform the firmware update.
+   See :ref:`app_build_fota` for more information about update files.
```

Comment:
I also think that we should update the table in the Multi-Image documentation:

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/multi_image.html